ibm/data_source_ibm_is_security_group_rules_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccIBMIsSecurityGroupRulesDataSourceBasic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccCheckIBMIsSecurityGroupRulesDataSourceConfigBasic(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("data.ibm_is_security_group_rules.is_security_group_rules", "id"),
+					resource.TestCheckResourceAttrSet("data.ibm_is_security_group_rules.is_security_group_rules", "security_group_id"),
+					resource.TestCheckResourceAttrSet("data.ibm_is_security_group_rules.is_security_group_rules", "rules.#"),
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckIBMIsSecurityGroupRulesDataSourceConfigBasic() string {
+	return fmt.Sprintf(`
+		data "ibm_is_security_group_rules" "is_security_group_rules" {
+			security_group_id = "r134-ebe8a7ce-16c5-42bb-bebc-bc4b753a8a7c"
```

Comment:
do not hard code IDs

Reply:
done - Interpolation added:
security_group_id = ibm_is_security_group.testacc_security_group.id